lib/ui/card.tsx

```diff
@@ -33,25 +44,71 @@ CardHeader.displayName = "CardHeader"
 const CardTitle = React.forwardRef<
   HTMLParagraphElement,
   React.HTMLAttributes<HTMLHeadingElement>
+>(({ className, ...props }, ref) => (
+  <h3 ref={ref} className={cn("text-base font-medium", className)} {...props} />
+))
+CardTitle.displayName = "CardTitle"
+
+const CardSubtitle = React.forwardRef<
+  HTMLParagraphElement,
+  React.HTMLAttributes<HTMLHeadingElement>
 >(({ className, ...props }, ref) => (
   <h3
     ref={ref}
-    className={cn(
-      "text-2xl font-semibold leading-none tracking-tight",
-      className
-    )}
+    className={cn("text-base font-normal text-muted-foreground", className)}
     {...props}
   />
 ))
-CardTitle.displayName = "CardTitle"
+CardSubtitle.displayName = "CardSubtitle"
+
+const CardInfo = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => (
+  <div ref={ref} className={cn("-ml-1 h-6 w-6", className)}>
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger
+          className="cursor-default text-muted-foreground"
+          title="Info"
+        >
+          <Icon icon={InfoCircleLine} size="md" />
+        </TooltipTrigger>
+        <TooltipContent>
+          <p {...props} />
+        </TooltipContent>
+      </Tooltip>
+    </TooltipProvider>
+  </div>
+))
+CardInfo.displayName = "CardInfo"
+
+const CardLink = React.forwardRef<
+  HTMLAnchorElement,
+  React.ComponentPropsWithoutRef<"a">
+>(({ className, ...props }) => {
+  return (
+    <a
+      className={cn(
+        "h-6 w-6 text-muted-foreground transition-colors hover:text-foreground",
+        className
+      )}
+      aria-label="Link"
+      {...props}
+    >
+      <Icon icon={ArrowRight} size="md" />
+    </a>
+  )
+})
+CardLink.displayName = "CardLink"
 
 const CardDescription = React.forwardRef<
   HTMLParagraphElement,
   React.HTMLAttributes<HTMLParagraphElement>
 >(({ className, ...props }, ref) => (
   <p
     ref={ref}
-    className={cn("text-sm text-muted-foreground", className)}
+    className={cn("px-4 text-sm text-muted-foreground", className)}
```

Comment:
This `PX` looks a bit suspicious to me. It almost looks as if `description` should live inside of `CardHeader` but we're bringing it outside just to add padding back again.

I would consider putting everything inside of CardHeader or removing `description` altogether, no?